client/v1/client.go

```diff
@@ -145,10 +145,9 @@ func New(conf Config) (*Client, error) {
 		}
 	}
 
-	tr := &http.Transport{
-		TLSClientConfig: &tls.Config{
-			InsecureSkipVerify: conf.InsecureSkipVerify,
-		},
+	tr := http.DefaultTransport.(*http.Transport)
+	tr.TLSClientConfig = &tls.Config{
```

Comment:
Since the DefaultTransport is a pointer aren't we modifying it globally? I think that will create a race condition as well.